apparmor.d/groups/apps/discord

```diff
@@ -40,5 +48,12 @@ profile discord @{exec_path} {
 
   owner @{run}/user/@{uid}/discord-ipc-@{int} rw,
 
+  owner @{user_config_dirs}/discord/** rwm,
+
+  @{PROC}/@{pid}/task/@{tid}/status r,
```

Comment:
Both lines are already included in `abstractions/common/electron`

Reply:
Just noticed the `m` in `rwm`. This is not in `abstractions/common/electron`.

There is no way we allow read write and memory mapping on all config dirs. Can you double check what files require it?
For example, in spotify, `rwm` is needed for WidevineCdm: https://github.com/roddhjav/apparmor.d/blob/ad960d477b8037005493fa1ca591d61d4f7936d5/apparmor.d/profiles-s-z/spotify#L41-L42

Reply:
There seem to be multile mudule files that require this.
The best I could manage is this:

```
deny owner @{user_config_dirs}/discord/0.0.54/modules/** w,
owner @{user_config_dirs}/discord/0.0.54/modules/** rm,
```

Reply:
Changes added as requested.

> we can only ensure `rwm` is limited to the minimum. 
Maybe off topic, but isn't this a security issue in discord by modern standards?

Reply:
Yes it is. Meanwhile it also depends how it is handled by discord. JIT compilation can be safe if they run in a sandbox (like in chromium).